src/features/generic/time-utils.ts

```diff
@@ -56,6 +56,31 @@ namespace TimeUtilities {
       displayInternal: displayInternal,
     }
   }
-};
+
+  export function calculateWorkedTimeAndBillableHours(user: TimeEntryTotalDto | DailyCombinedData){
+
+    // calculates billable hours and adds percentage sign
+    const billableHours = (user.projectTime/user.expected * 100).toFixed(1); 
+    const billableHoursWithPercentage = billableHours.concat("%");
```

Comment:
Use string literal syntax in return instead of concating two strings

Reply:
Done